packages/web-components/src/components/ic-button/ic-button.tsx

```diff
@@ -64,6 +64,11 @@ export class Button {
   @State() describedByContent: string = null;
   @State() title: string = null;
 
+  /**
+   * If 'fileUpload' is set to 'true', this is the accepted list of files.
```

Comment:
super minor: 'accepted list of files' - could you clarify that this is filetypes please. Sorry i didn't pick this up on first pass.

Reply:
@zxh722 have you thought about this? I agree with @GCHQ-Developer-299. Makes sense to clarify :)

Reply:
Also just a tiny thing from me about this prop description - would be good to use backticks for consistency with other descriptions (same with line 213)
```suggestion
   * If 'fileUpload' is set to `true`, this is the accepted list of files.
```